packages/jest-reporters/src/index.ts

```diff
@@ -14,12 +14,14 @@ import {
   trimAndFormatPath,
 } from './utils';
 
-export type {Config} from '@jest/types';
 export type {
   AggregatedResult,
-  SnapshotSummary,
```

Comment:
Fixed. Types for all public classes and utilities are exported. `SnapshotSummary` type is only used by `getSnapshotSummary()` util. This one is not exposed, but perhaps it should be exported together with `getSnapshotStatus()`? Looks helpful.